src/main/java/com/google/sps/servlets/HistoryServlet.java

```diff
@@ -48,6 +48,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         // Get the id of the student whose tutoring history will be displayed.
         // Make the default id -1 if the parameter is null, so an empty list will be returned
         String studentID = Optional.ofNullable((String)request.getSession(false).getAttribute("userId")).orElse("-1");
+        String studentIDTutorView = Optional.ofNullable(request.getParameter("studentIDTutorView")).orElse("-1");
```

Comment:
What is this used for?

Reply:
This is only passed down if the person trying to see the student's history is the student's tutor. If that is not the case, then the servlet will just load the history of the currently logged in user.